src/Sarif/Writers/SarifLogger.cs

```diff
@@ -86,13 +86,11 @@ private static string Redact(string text, IEnumerable<string> tokensToRedact)
 
         public SarifLogger(
             string outputFilePath, 
-            bool verbose,
-            bool computeTargetsHash,
-            Tool tool, 
-            Run run)
+            LoggingOptions loggingOptions = LoggingOptions.PrettyPrint,
```

Comment:
>LoggingOptions.PrettyPrint [](start = 44, length = 26)

Consider defining, in this class:
```
private const LoggingOptions DefaultLoggingOptions = LoggingOptions.PrettyPrint;
```
Then if you change your mind about the default you can change it in one place.

Reply:
nice.

---
In reply to: [135840849](https://github.com/Microsoft/sarif-sdk/pull/609#discussion_r135840849) [](ancestors = 135840849)